common/head_tag.html

```diff
@@ -28,152 +28,152 @@
               <h2 id="trending-guides" class="col-header">{{theme-setting 'footer_columns_header'}}</h2>              
               <ul aria-labelledby="trending-guides" class="trending-guides-articles">
                 <li>
-                  <a href={{theme-setting 'article0link'}}>
+                  <a href={{theme-setting 'article0link'}} rel="noopener noreferrer" target="_blank" >
```

Comment:
```suggestion
                  <a href={{theme-setting 'article0link'}} target="_blank" >
```

I believe we do actually want the referrer data here. It points to a site we own, and the analytics are helpful.

Reply:
Okay you're the boss